app/src/main/java/com/example/gotproject/DataCharactersItem.kt

```diff
@@ -1,20 +1,12 @@
 package com.example.gotproject
 
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "DataCharacters")
```

Comment:
Nitpick: I recommend using a different model for networking than for persistence with a simple mapper between them. It can make your life much easier and you don't mix annotations types.